src/tests/App.test.tsx

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 import * as ReactDOM from 'react-dom';
-import App from './App';
+import App from '../react_apps/App';
```

Comment:
I'm wondering if this is the best place for the `tests` folder - it might make more sense to have it alongside the rest of the  `components` (see above). Maybe like:
```
src
-----components
----------App
---------------App.tsx
---------------App.test.tsx
----------SomeOtherComponent
---------------SomeOtherComponent.tsx
---------------SomeOtherComponent.test.tsx
```
or maybe
```
src
-----components
----------App.tsx
----------SomeOtherComponent.tsx
-----tests
----------App.test.tsx
----------SomeOtherComponent.test.tsx
```
I don't have a huge preference here, but it's something to think about.

Reply:
I don't know either.
Thurday I think we should figure out a standard that the team should follow